core/src/test/scala/org/apache/spark/metrics/source/AccumulatorSourceSuite.scala

```diff
@@ -37,7 +36,7 @@ class AccumulatorSourceSuite extends SparkFunSuite {
     val accs = Map("my-accumulator-1" -> acc1,
                    "my-accumulator-2" -> acc2)
     LongAccumulatorSource.register(mockContext, accs)
-    val captor = new ArgumentCaptor[AccumulatorSource]()
+    val captor = ArgumentCaptor.forClass(classOf[AccumulatorSource])
```

Comment:
This is actually just cleaning up a deprecation warning along the way; not strictly required for Java 11